ParseCSV/ParseCSV/Table/MyTable.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ParseCSV
+{
+    class MyTable
+    {
+        public List<Row> Table;
+
+        public MyTable()
+        {
+            Table = new List<Row>();
+        }
+
+        public Row this[int index]
+        {
+            get
+            {
+                return Table[index];
+            }
+
+            set
+            {
+                Table.Insert(index, value);
```

Comment:
What if we want to modify existing entry? with current setter it's not possible, it will insert new entry

Reply:
No, it works. It will work, like you wrote, if I will make:  Table.**Add**(index, value);

Reply:
Normally setter should **update** existing entry (which is found by index), and if it doesn't exist - create it ( aka "AddOrUpdate" behavior)